Scripts/Runtime/Entities/Tasks/Data/Wrapper/VDWrapperForAdd.cs

```diff
@@ -0,0 +1,24 @@
+using Anvil.Unity.DOTS.Data;
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    internal class VDWrapperForAdd : AbstractVDWrapper
+    {
+        public VDWrapperForAdd(AbstractVirtualData data) : base(data)
+        {
+        }
+
+        public override JobHandle Acquire()
+        {
+            Data.AccessController.Acquire(AccessType.SharedWrite);
+            return default;
+        }
+
+        public override void Release(JobHandle releaseAccessDependency)
+        {
+            Data.AccessController.Release();
+        }
```

Comment:
I had originally split them up because I didn't want to have any if checks for whether we were acquiring async or directly. 

But your comment spurred some investigation and yeah, we already know the context to call so I've gotten rid of half the wrappers and all wrappers have an async and direct path.

Reply:
Agreed but no longer relevant since we implemented a `Async` and `non-Async` path.